### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by KrisLanser**
Arduino code om gps coordinaten te versturen

### CODE REVIEW

File: arduinoCode/gpsLoraCode/gpsTransmit.ino [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#include <TheThingsNetwork.h>
+
+// Set your AppEUI and AppKey
+const char *appEui = "70B3D57ED0022E0E";
+const char *appKey = "409945AA983522B340F08BAEAD0C1529";
+
+#define loraSerial Serial1
+#define debugSerial Serial
+
+// Replace REPLACE_ME with TTN_FP_EU868 or TTN_FP_US915
```
Comment (BenjaminKnol) [inline arduinoCode/gpsLoraCode/gpsTransmit.ino]: pliss delet

File: arduinoCode/gpsLoraCode/gpsTransmit.ino [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+//    debugSerial.println("-- JOIN");
+//    ttn.join(appEui, appKey);
+}
+
+void loop()
+{
+  debugSerial.println("-- LOOP");
+
+  double LONG = 56.234464;
+  double LANG = 23.345908;
```
Comment (BenjaminKnol) [inline arduinoCode/gpsLoraCode/gpsTransmit.ino]: rename var to some sensible abreviation of lattitude and longitutde don't use long as it is a datatype. Also capitalization is usually used when creating finals ergo a var that doesn't change.

File: arduinoCode/gpsLoraCode/gpsTransmit.ino [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  // Prepare payload of 1 byte to indicate LED status
```
Comment (BenjaminKnol) [inline arduinoCode/gpsLoraCode/gpsTransmit.ino]: this is a fucking lie

File: arduinoCode/gpsLoraCode/gpsTransmit.ino [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  payload.f[0] = LONG;
+  payload.f[1] = LANG;
+
+  ttn.sendBytes(payload.bytes, sizeof(payload));
+  for (int i = 0; i < 8; i++) {
+    debugSerial.print(payload.bytes[i], HEX);
+    debugSerial.print(" ");
+  }
+
+  delay(10000);
```
Comment (BenjaminKnol) [inline arduinoCode/gpsLoraCode/gpsTransmit.ino]: find out if you can more elegantly indicate a delay as working in milliseconds is silly when we want to have delays of 10 minutes

Comment (KrisLanser) [reply arduinoCode/gpsLoraCode/gpsTransmit.ino]: Niet zo zeiken

**Pull request closed without merging**